src/libs/actions/OnyxUpdateManager.ts

```diff
@@ -41,11 +41,18 @@ export default () => {
     Onyx.connect({
         key: ONYXKEYS.ONYX_UPDATES_FROM_SERVER,
         callback: (value) => {
-            // When the OpenApp command hasn't finished yet, we should not process any updates.
-            if (!value || isLoadingApp) {
-                if (isLoadingApp) {
-                    console.debug(`[OnyxUpdateManager] Ignoring Onyx updates while OpenApp hans't finished yet.`);
-                }
+            // When there's no value, there's nothing to process, so let's return early.
+            if (!value) {
+                return;
+            }
+            // If isLoadingApp is positive it means that OpenApp command hasn't finished yet, and in that case
+            // we don't have base state of the app (reports, policies, etc) setup. If we apply this update,
+            // we'll only have them overriten by the openApp response. So let's skip it and return.
+            if (isLoadingApp) {
+                // When ONYX_UPDATES_LAST_UPDATE_ID_APPLIED_TO_CLIENT is set, we pause the queue. Let's unpause
```

Comment:
Hm, I still can't see in the code where the queue is being paused where [this is set.](https://github.com/Expensify/App/blob/ad1a1a6860cc5126e36d59a0837b54ff33774572/src/libs/actions/OnyxUpdates.ts#L106)